src/FoodTrans.Auth.Application/Users/Commands/LoginCommand.cs

```diff
@@ -0,0 +1,36 @@
+using Application.Contracts;
+using Domain.User;
+using Domain.User.ValueObjects;
+using ErrorOr;
+using MediatR;
+
+namespace Application.Users.Commands;
+
+public sealed class LoginCommand : IRequest<ErrorOr<User>>
+{
+    public string Email { get; set; }
+    public string Username { get; set; }
+    public string Password { get; set; }
+}
+
+public sealed class LoginCommandHandler : IRequestHandler<LoginCommand, ErrorOr<User>>
+{
+    private readonly IUserRepository _userRepository;
+
+    public LoginCommandHandler(IUserRepository userRepository)
+    {
+        _userRepository = userRepository;
+    }
+
+    public async Task<ErrorOr<User>> Handle(LoginCommand request, CancellationToken cancellationToken)
+    {
+        var email = Email.Create(request.Email);    // TODO: Niepotrzebne tworzenie obiektu VO (walidacja jest zbędna przy Query?)
+
+        if (email.IsError)
+        {
+            return Error.Conflict();    // TODO: Zwrócenie niepoprawnego błędu - powinien dotyczyć adresy email
+        }
```

Comment:
Tutaj nie jestem w stanie CI powiedzieć, bo nie wiem jak przebiega flow, ale jeżeli metoda Create zwróci Ci obiekt z proprty isError w momencie nie znalezienia, to chyba okej

Reply:
Tak. Tutaj zwracany jest obiekt generyczny `ErrorOr<T>`. W nim zawierają się m.in. `IsError`, `Errors` oraz `Value` <- rzeczywista wartość, czyli tutaj obiekt np. `Email`.

Reply:
Tutaj masz kod tej klasy `ErrorOr<>`:
https://github.com/amantinband/error-or/blob/main/src/ErrorOr.cs

Reply:
No spoko, workable :D